test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -51,6 +56,20 @@
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
+  @Before
```

Comment:
previous to this CL every test had its own test-localized data.

after this CL and part2 and part3 the plan is to have tables/data for the tests ready before any of them start?

not a deal breaker, just asking.